.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ test-ci ]
```

Comment:
Do you want to change this to develop branch before commit or is it going to a reference commit on a test_ci branch

Reply:
Good catch!  Will update it.